advanced-creation.md

```diff
@@ -111,3 +111,49 @@ const unicorn = got.create(settings);
 // Same as:
 const unicorn = got.extend({headers: {unicorn: 'rainbow'}});
 ```
+
+### Merging instances
+
+#### got.merge(instances, [methods])
```

Comment:
`.merge()` is too ambiguous. Let's name it `.mergeInstances()`.

Reply:
Verbose is better than unclear. It's only a few extra characters and you don't write it that often.

Reply:
Okay, I'll change it then.

Reply:
done.